authentication/understanding-identity-provider.adoc

```diff
@@ -72,6 +72,12 @@ link:http://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth[Authoriza
 Once an identity provider has been defined, you can
 xref:../authentication/using-rbac.adoc#authorization-overview_using-rbac[use RBAC to define and apply permissions].
 
+[NOTE]
+====
+When configuring multiple identity providers, some of them, such as `Request header` and `HTPasswd` can not be
```

Comment:
"some" indicates ambiguity. suggestion: we should be explicit which ones cannot coexist.

Reply:
Moved to the other section and clarified.